tests/core/test_rule_groups.py

```diff
@@ -0,0 +1,38 @@
+from smdebug_rulesconfig.debugger_rules._constants import (
+    SUPPORTED_DL_FRAMEWORKS,
+    SUPPORTED_FRAMEWORKS,
+)
+from smdebug_rulesconfig.debugger_rules._ruleGroups import (
+    DEEP_LEARNING_APPLICATION_RULES,
+    DEEP_LEARNING_RULES,
+    UNIVERSAL_RULES,
+    XGBOOST_RULES,
+)
+from smdebug_rulesconfig.debugger_rules._utils import _get_rule_list
+
+
+def test_universal_rules():
+    for rule in UNIVERSAL_RULES:
+        for framework in SUPPORTED_FRAMEWORKS:
+            if rule not in _get_rule_list(framework):
+                assert False, f"Rule: {rule} not supported by {framework}"
```

Comment:
nit: assert on the condition you want, instead of checking for the opposite and asserting False.

Reply:
same applies for tests below.

Reply:
Done